Command Pattern/Command Pattern/LightOffCommand.cpp

```diff
@@ -0,0 +1,15 @@
+//
```

Comment:
I'm on the fence for this one. I don't want to teach the habit of creating header files just because a source file is more work, but I agree in this case, a single convenience subclass constructor seems excessive.